### ISSUE TITLE
Include Xcode project file in repo to make contribution easier

### ISSUE THREAD

**Issue by yoshisuga** [ckosmic/sm64ex-ios#4]
Great work on this and thank you for including easy-to-understand instructions! Amazing to see this run natively on my iPhone!

It'd be great if the Xcode project file was part of the repo so that one can build, run, debug and easily contribute any iOS specific code. I know it takes adding the sources to the project and the flags from the makefile to the Build Settings - I wonder if there's an easier way to do that. I know cmake has an Xcode generator that other projects like PPSSPP use, but having an Xcode project file already in the repo would be a nice to have.

Thanks again!!

**Comment by dlevi309** [comment]
It doesn't use the Xcode project for compilation though, the point of creating the project is to 1. create a provisioning profile, and 2. generate blank storyboards that are then linked when building the ipa

**Comment by yoshisuga** [comment]
> It doesn't use the Xcode project for compilation though, the point of creating the project is to 1. create a provisioning profile, and 2. generate blank storyboards that are then linked when building the ipa

Yes, I understand that a project is created just for provisioning profile, but this is for basically moving the tasks in the makefile to the Xcode project as steps in the build phases (to extract assets first, etc).

**Comment by ckosmic** [comment]
The plan is to move everything to an Xcode project eventually.  I'll probably either do an "External Build System" type project or convert it using Cmake.  Cmake sounds like the better option, but I'd have to see how that works first.

**Comment by manilynlumalang0306** [comment]
i see this app

**Comment by ckosmic** [comment]
The Xcode project is coming along fairly nicely so far.  I've pushed the changes to the testing branch, however I need to test it on my painfully slow MacOS VM before merging it, but it should work as expected.  I went ahead with an approach that runs the build script on build in Xcode which allows it to automatically target iOS devices/simulators directly, sign and debug as well.  Pretty nice compared to terminal commands haha.
